packages/extension_google_sign_in_as_googleapis_auth/example/test/widget_test.dart

```diff
@@ -0,0 +1,29 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// This is a basic Flutter widget test.
+//
+// To perform an interaction with a widget in your test, use the WidgetTester
+// utility that Flutter provides. For example, you can send tap and scroll
+// gestures. You can also use WidgetTester to find child widgets in the widget
+// tree, read text, and verify that the values of widget properties are correct.
+
+import 'package:extension_google_sign_in_example/main.dart';
+import 'package:flutter/material.dart';
+
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('App starts smoke test', (WidgetTester tester) async {
+    // Build our app and trigger a frame.
+    await tester.pumpWidget(
+      const MaterialApp(
+        title: 'Google Sign In',
+        home: SignInDemo(),
+      ),
+    );
+
+    expect(find.byKey(signInButtonKey), findsOneWidget);
+  });
+}
```

Comment:
Remove this file

Reply:
I'll also clean up the Key I added for that locator :D